frontend/src/slices/articles/articles.slice.ts

```diff
@@ -0,0 +1,37 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import { DataStatus } from '~/libs/enums/enums.js';
+import { type ValueOf } from '~/libs/types/types.js';
+import { type ArticleBaseResponseDto } from '~/packages/articles/articles.js';
+
+import { createArticle } from './actions.js';
+
+type State = {
+  articles: ArticleBaseResponseDto[];
+  dataStatus: ValueOf<typeof DataStatus>;
+};
+
+const initialState: State = {
+  articles: [],
+  dataStatus: DataStatus.IDLE,
+};
+
+const { reducer, actions, name } = createSlice({
+  initialState,
+  name: 'articles',
+  reducers: {},
+  extraReducers(builder) {
+    builder.addCase(createArticle.pending, (state) => {
+      state.dataStatus = DataStatus.PENDING;
+    });
+    builder.addCase(createArticle.fulfilled, (state, action) => {
+      state.articles.push(action.payload);
```

Comment:
I guess redux-toolkit's immer will deal with it, but let's make it not in that rough way
```suggestion
      state.articles = state.articles.concat(action.payload);
```